allennlp/training/trainer.py

```diff
@@ -33,6 +34,7 @@ def __init__(self,
                  cuda_device: int = -1,
                  grad_norm: Optional[float] = None,
                  grad_clipping: Optional[float] = None,
+                 learning_rate_scheduler: Optional[torch.optim.lr_scheduler._LRScheduler] = None,  # pylint: disable=protected-access
```

Comment:
It'd probably be a bit nicer to just do `from torch.optim.lr_scheduler import _LRScheduler as LRScheduler` at the top of the file, so this line isn't as ugly.